upgrade_everything_v1.5.2/data/upgrade-everything/recipes/wooden-to-stone-hoe.json

```
@@ -1,12 +1,12 @@
-{
-	"type": "minecraft:smithing",
-	"base": {
-		"item": "minecraft:wooden_hoe"
-	},
-	"addition": {
-		"item": "minecraft:cobblestone"
-	},
-	"result": {
-		"item": "minecraft:stone_hoe"
-	}
-}
+{
+	"type": "minecraft:smithing",
+	"base": {
+		"item": "minecraft:wooden_hoe"
+	},
+	"addition": {
+		"item": "minecraft:stone"

```

I'd prefer cobblestone to stay used here, due to cobblestone being what is used to craft stone toosl in the first place
```suggestion
                "item": "minecraft:cobblestone"
```